AsyncDisplayKit/Layout/ASRelativePositionLayoutSpec.h

```diff
@@ -0,0 +1,57 @@
+//
+//  ASRelativePositionLayoutSpec.h
+//  Pods
+//
+//  Created by Samuel Stow on 12/31/15.
+//
+//
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+/** How the child is positioned within the spec. */
+typedef NS_OPTIONS(NSUInteger, ASRelativePositionLayoutSpecPosition) {
+    /** The child is positioned at point 0 relatively to the layout axis (ie left / top most) */
+    ASRelativePositionLayoutSpecPositionZero = 0,
+    /** The child is centered along the specified axis */
+    ASRelativePositionLayoutSpecPositionCenter = 1 << 0,
+    /** The child is positioned at the maximum point of the layout axis (ie right / bottom most) */
+    ASRelativePositionLayoutSpecPositionMax = 1 << 1,
+};
+
+/** How much space the spec will take up. */
+typedef NS_OPTIONS(NSUInteger, ASRelativePositionLayoutSpecSizingOption) {
+    /** The spec will take up the maximum size possible */
+    ASRelativePositionLayoutSpecSizingOptionDefault,
+    /** The spec will take up the minimum size possible along the X axis */
+    ASRelativePositionLayoutSpecSizingOptionOptionMinimumX = 1 << 0,
```

Comment:
Typo: "Option" is repeated twice :)

I've been bothered by the naming convention of sizing option in center spec since the beginning. Only that spec (and now this new one) uses the notion of "X" and "Y". Maybe we should take this chance to replace them with "width" and "height" instead, i.e "MinimumWidth" and "MinimumHeight". Also, "MinimumXY" can be renamed to "MinimumSize". What do you think?